OpenAI-DotNet/Proxy/OpenAIProxyStartup.cs

```diff
@@ -103,11 +103,16 @@ private async Task HandleRequest(HttpContext httpContext, string endpoint)
 
                 foreach (var header in proxyResponse.Headers)
                 {
+                    // transfer encoding (and specifically 'chunked') occurs when different character sets are used - for example, latin and greek
+                    // the client doesn't know how to handle it, and crashes
+                    // when removed, it works just fine ..
+                    if (header.Key == "Transfer-Encoding") continue;
```

Comment:
can we add this to a pre-defined readonly collection?

Reply:
This is a good reference: 
https://github.com/microsoft/reverse-proxy/blob/51d797986b1fea03500a1ad173d13a1176fb5552/src/ReverseProxy/Forwarder/RequestUtilities.cs#L61-L83

Reply:
https://github.com/microsoft/reverse-proxy/tree/51d797986b1fea03500a1ad173d13a1176fb5552/src/ReverseProxy/Forwarder